tests/compat-tests.el

```diff
@@ -2,131 +2,132 @@
 
 ;; Copyright (C) 2022  Free Software Foundation, Inc.
 
-(require 'poly-markdown nil t)          ;Don't fail if not installed.
-
-(define-innermode poly-test-markdown-inline-python-innermode  poly-markdown-inline-code-innermode
-  :head-matcher (cons "[^`]\\(`py \\)" 1)
-  :mode 'python-mode
-  :fallback-mode 'poly-fallback-mode
-  :tail-mode nil
-  :head-mode nil)
-
-(define-polymode poly-test-markdown-mode poly-markdown-mode
-  :innermodes `(:inherit
-                poly-test-markdown-inline-python-innermode))
-
-(ert-deftest compat/lsp/lsp-text ()
-  (skip-unless (fboundp 'poly-markdown-mode))
-  (let ((markdown-enable-math t))
+(when (require 'poly-markdown nil t)          ;Don't fail if not installed.
+
```

Comment:
I am not entirely comfortable with this conditional. it will surely lead to confusion at some point, either locally or in the CI.

Reply:
Restore to the original code